src/analyze/analyze.c

```diff
@@ -2214,6 +2227,8 @@ static int help(int argc, char *argv[], void *userdata) {
                "     --threshold=N           Exit with a non-zero status when overall\n"
                "                             exposure level is over threshold value\n"
                "     --version               Show package version\n"
+               "     --json-path             Obtain a set of custom security requirements\n"
```

Comment:
this takes an argument, should be suffixed with `=PATH` or so

Reply:
please let's call this "security policy" or so, and stick to it, i.e. not "security requirements"